app/views/users/new.html.erb

```diff
@@ -1,32 +1,10 @@
-<% provide :title, t("sign_up") %>
-<h1><%= t("sign_up") %></h1>
+<% provide :title, t(:sign_up) %>
+<% provide(:button_text, t(:create_my_account)) %>
+<h1><%= t(:sign_up) %></h1>
 <div class="row">
   <div class="col-md-6 col-md-offset-3">
     <%= form_for @user, url: signup_path do |f| %>
```

Comment:
remove this file, because you have render form partial after at line 7

Reply:
hi i already editted this file, but after i push, it didnt show in github
![image](https://user-images.githubusercontent.com/87749692/131635741-92973c51-f3f7-4928-999b-47693db0d805.png)